librz/core/cmd_api.c

```diff
@@ -537,7 +540,7 @@ static void get_minmax_argc(RzCmdDesc *cd, int *min_argc, int *max_argc) {
 }
 
 static RzOutputMode cd_suffix2mode(RzCmdDesc *cd, const char *cmdid) {
-	if (cd->type != RZ_CMD_DESC_TYPE_ARGV_MODES) {
+	if (cd->type != RZ_CMD_DESC_TYPE_ARGV_MODES && cd->type != RZ_CMD_DESC_TYPE_ARGV_STATE) {
```

Comment:
This long condition repeats multiple time, what about creating a macro or simple inline function for it?